docs/api.rst

```diff
@@ -1372,6 +1385,24 @@ newClockTimer
 
 Create a new :ref:`Timer` that uses the provided :ref:`Clock` as a source of the current :ref:`Time`.
 
+.. _newVirtualTimer:
+
+newVirtualTimer
+```````````````
+
+.. code-block:: haskell
+
+  newVirtualTimer :: () -> VirtualTimer
+
+Create a new :ref:`VirtualTimer`. You progress its time manually, i.e., its time doesn’t progress automatically. Use it for example for testing.
+
+.. code-block:: javascript
+
+  const timer = newVirtualTimer()
+  console.log(timer.now()) // 0
+  timer.tick(10);
```

Comment:
Very minor: remove semicolon.

Anyone know of a way to lint code samples in rST?